pkg/server/server_test.go

```diff
@@ -140,7 +140,7 @@ func TestCreateServer(t *testing.T) {
 	// }, 500*time.Millisecond, 50*time.Millisecond)
 
 	require.Eventually(t, func() bool {
-		q2, err := client1.QueryEnvelopes(ctx, &message_api.QueryEnvelopesRequest{
+		q2, err := client2.QueryEnvelopes(ctx, &message_api.QueryEnvelopesRequest{
```

Comment:
(This was a mistake in the original unit test - it wasn't actually testing the peering without this change)